x/farming/keeper/reward.go

```diff
@@ -18,29 +20,9 @@ func (k Keeper) GetReward(ctx sdk.Context, stakingCoinDenom string, farmerAcc sd
 	return reward, true
 }
 
-// GetAllRewards returns all rewards in the Keeper.
-func (k Keeper) GetAllRewards(ctx sdk.Context) (rewards []types.Reward) {
-	k.IterateAllRewards(ctx, func(reward types.Reward) (stop bool) {
-		rewards = append(rewards, reward)
-		return false
-	})
-
-	return rewards
-}
-
-// GetRewardsByStakingCoinDenom reads from kvstore and return a specific Reward indexed by given staking coin denom
-func (k Keeper) GetRewardsByStakingCoinDenom(ctx sdk.Context, denom string) (rewards []types.Reward) {
-	k.IterateRewardsByStakingCoinDenom(ctx, denom, func(reward types.Reward) bool {
-		rewards = append(rewards, reward)
-		return false
-	})
-
-	return rewards
-}
```

Comment:
It think `GetAllRewards` and `GetRewardsByStakingCoinDenom` will be used other core logics